sdk/core/core-amqp/src/connectionConfig/connectionConfig.ts

```diff
@@ -91,7 +91,7 @@ export namespace ConnectionConfig {
     }
 
     if (!parsedCS.Endpoint.endsWith("/")) parsedCS.Endpoint += "/";
-
+    parsedCS.Endpoint = parsedCS.Endpoint.replace(new RegExp("[^w]+://"), "sb://");
```

Comment:
I'm pretty sure `[^w]` means any character that is not a `w`.

Reply:
Oh, definitely missed that, thank you so much. :)
Updated again, please check.